src/Domain/DomainCreators/RegisterDerivedWithCharm.cpp

```diff
@@ -0,0 +1,25 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/CoordinateMaps/AffineMap.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.hpp"
+#include "Domain/DomainCreators/Interval.hpp"
+#include "Parallel/CharmPupable.hpp"
+
+namespace DomainCreators {
+void register_derived_with_charm() {
+  PUPable_reg(SINGLE_ARG(::CoordinateMap<Frame::Logical, Frame::Inertial,
```

Comment:
I'm thinking of how to keep this list maintainable in the long run. We should probably have functions for 1D, 2D and 3D. Alphabetical by map name, left to right within each function?